### DISCUSSION TITLE
Error trying to create App ID

### DISCUSSION THREAD

**Discussion by cshield-ez** [InboxSDK/InboxSDK#944] [Q&A] [answered]
Hey everyone

Trying to register my InboxSDK app - am I using an incorrect format? Can't find any documentation anywhere, used a variety of strings (including my chrome ext. ID), numbers and other things but hit with the following error when I click Create AppID on https://www.inboxsdk.com/register

"Error creating AppId:
Invalid app id seed"

I'm also definitely logged into my InboxSDK account. Anyone know what's causing the errors?

**Comment by aleemstreak** [comment, marked as answer]
The AppId needs to be an ascii alpha numeric string 5 to 15 characters long with no spaces or punctuation.
ᐧ

**Reply by cshield-ez** [reply]
Somehow everything i've tried didn't manage to match something so simple, thanks Aleem. Very little experience coding but seeing if I can team up with generative AI to make some local chrome extensions.

**Reply by valentinkorzun** [reply]
@aleemstreak Sorry for the off-topic post, but could you tell me why registering an app ID is required? I haven't been able to locate this information.

**Comment by aleemstreak** [comment]
We want to be able to email active developers if there is a Gmail change
and updating your version of the SDK is urgently needed. We have telemetry
that tells us which extensions are on which versions of the SDK.
ᐧ
